site/docs/integrations/esbuild.md

```diff
@@ -100,6 +100,7 @@ Different formatting of identifiers (e.g. class names, keyframes, CSS Vars, etc)
 
 - `short` identifiers are a 7+ character hash. e.g. `hnw5tz3`
 - `debug` identifiers contain human readable prefixes representing the owning filename and a potential rule level debug name. e.g. `myfile_mystyle_hnw5tz3`
+- A function accepting three parameters: A scope name, an index, and an optional rule level debug name, and returning the identifier.
```

Comment:
Can you update this and the other docs to reference the new API.

Reply:
Done